core/metamodel/src/main/java/org/apache/isis/core/metamodel/adapter/oid/Oid.java

```diff
@@ -59,16 +59,15 @@
     boolean isViewModel();
 
     boolean isPersistent();
+    
+    default boolean isValue() {
+        return false;
+    }
```

Comment:
ObjectAdapters require a non-null Oid field now -> hence Oid provides info whether it represents a value; 
if Oid represents a value, then the Oid is just a singleton constant immutable RootOid object, defined in RootOid.